File: Ioana_Hisham/src/main/java/ql/ast/expressions/Binary/LessThanOrEqual.java
package ql.ast.expressions.Binary;

import ql.ast.expressions.Expression;

public class LessThanOrEqual extends Binary {

    public LessThanOrEqual(int lineNumber, Expression leftHandSide, Expression rightHandSide) {
        super(lineNumber, leftHandSide, rightHandSide);
    }
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QL/AST/Expression/LessThanOrEqual.cs
namespace Assignment1.Model.QL.AST.Expression
{
    public class LessThanOrEqual : Binary
    {
        public LessThanOrEqual(int lineNumber, IExpression left, IExpression right) : base(left, right)
        {
            _lineNumber = lineNumber;
        }

        public override void Accept(IExpressionVisitor visitor) => visitor.Visit(this);
    }
}